public/app/features/dashboard-scene/sharing/ShareExportTab.tsx

```diff
@@ -0,0 +1,200 @@
+    saveAs(blob, `${dashboardJson.title}-${time}.json`);
+    trackDashboardSharingActionPerType('save_export', shareDashboardType.export);
+  }
+}
+
+function ShareExportTabRenderer({ model }: SceneComponentProps<ShareExportTab>) {
+  const { isSharingExternally, shouldTrimDefaults, isViewingJSON, modalRef } = model.useState();
+
+  const dashboardJson = useAsync(async () => {
+    if (isViewingJSON) {
+      const json = await model.getExportableDashboardJson();
+      return JSON.stringify(json, null, 2);
+    }
+
+    return '';
+  }, [isViewingJSON]);
+
+  return (
+    <>
+      {!isViewingJSON && (
+        <>
+          <p className="share-modal-info-text">
+            <Trans i18nKey="share-modal.export.info-text">Export this dashboard.</Trans>
+          </p>
+          <VerticalGroup spacing="md">
+            <Field label={exportExternallyTranslation}>
+              <Switch
+                id="share-externally-toggle"
+                value={isSharingExternally}
+                onChange={model.onShareExternallyChange}
+              />
+            </Field>
+
+            {config.featureToggles.trimDefaults && (
+              <Field label={exportDefaultTranslation}>
+                <Switch id="trim-defaults-toggle" value={shouldTrimDefaults} onChange={model.onTrimDefaultsChange} />
+              </Field>
+            )}
+          </VerticalGroup>
+
+          <Modal.ButtonRow>
+            <Button
+              variant="secondary"
+              onClick={() => {
+                modalRef?.resolve().onDismiss();
+              }}
+              fill="outline"
+            >
+              <Trans i18nKey="share-modal.export.cancel-button">Cancel</Trans>
+            </Button>
+            <Button variant="secondary" icon="brackets-curly" onClick={model.onViewJSON}>
+              <Trans i18nKey="share-modal.export.view-button">View JSON</Trans>
+            </Button>
+            <Button variant="primary" icon="save" onClick={() => model.onSaveAsFile()}>
+              <Trans i18nKey="share-modal.export.save-button">Save to file</Trans>
+            </Button>
+          </Modal.ButtonRow>
+        </>
+      )}
+
+      {isViewingJSON && (
+        <>
+          <AutoSizer disableHeight>
+            {({ width }) => {
+              if (dashboardJson.value) {
+                return (
+                  <CodeEditor
+                    value={dashboardJson.value ?? ''}
+                    language="json"
+                    showMiniMap={false}
+                    height="500px"
+                    width={width}
+                  />
+                );
+              }
+
+              if (dashboardJson.loading) {
+                return <div>Loading...</div>;
+              }
+
+              return null;
+            }}
+          </AutoSizer>
+
+          <Modal.ButtonRow>
+            <Button variant="secondary" fill="outline" onClick={model.onViewJSON} icon="arrow-left">
+              {/* TODO: Add translation */}
```

Comment:
@joshhunt - how to do that? :) I mean I know where to add it but i don't know the languages we are using. So the question is- should I just use ChatGPT to translate the string for me?

Reply:
Annotate the phrase correctly (with an ID) [according to our docs](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#tldr), run `yarn i18n:extract` like [CI tells you to](https://drone.grafana.net/grafana/grafana/137470/2/6) and commit it all 

Then they automatically get sucked up into crowdin, translated by our translators, and then [comitted back into our repo](https://github.com/grafana/grafana/pull/67043) :)

Reply:
Lovely, thank you! Sorry, I didn't put an effort into looking for https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#tldr, my bad!